packages/core/src/Stores/notification-store.js

```diff
@@ -922,6 +930,24 @@ export default class NotificationStore extends BaseStore {
                 message: <Localize i18n_default_text='Check your connection.' />,
                 type: 'danger',
             },
+            close_uk_account: {
```

Comment:
We sort these notifications alphabetically as well. Please, help to move it to the correct place in the file🙏🏻